gridstatus/miso.py

```diff
@@ -161,6 +163,87 @@ def _get_load_and_forecast_data(self, verbose=False):
         r = self._get_json(url, verbose=verbose)
         return r
 
+    forecast_cols = [
+        "Interval Start",
+        "Interval End",
+        "Publish Time",
+        "North",
+        "Central",
+        "South",
+        "MISO",
+    ]
+
+    @support_date_range(frequency="DAY_START")
+    def get_solar_forecast(self, date, verbose=False):
+        if date == "latest":
+            return self.get_solar_forecast(date="today", verbose=verbose)
+
+        return self._get_solar_and_wind_forecast_data(
+            date,
+            fuel="solar",
+            verbose=verbose,
+        )[self.forecast_cols]
+
+    @support_date_range(frequency="DAY_START")
+    def get_wind_forecast(self, date, verbose=False):
+        if date == "latest":
+            return self.get_wind_forecast(date="today", verbose=verbose)
+
+        return self._get_solar_and_wind_forecast_data(
+            date,
+            fuel="wind",
+            verbose=verbose,
+        )[self.forecast_cols]
+
+    def _get_solar_and_wind_forecast_data(self, date, fuel, verbose=False):
+        # Example url: https://docs.misoenergy.org/marketreports/20240327_mom.xlsx
+        url = f"https://docs.misoenergy.org/marketreports/{date.strftime('%Y%m%d')}_mom.xlsx"  # noqa
+
+        log(f"Downloading solar and wind forecast data from {url}", verbose)
+
+        try:
+            # Ignore the UserWarning from openpyxl about styles
+            warnings.filterwarnings("ignore", category=UserWarning, module="openpyxl")
+            excel_file = pd.ExcelFile(url, engine="openpyxl")
+        except urllib.error.HTTPError as e:
+            if e.status == 404:
+                raise NoDataFoundException(
+                    f"No solar or wind forecast found for {date}",
+                )
+
+        publish_time = pd.to_datetime(excel_file.book.properties.modified, utc=True)
```

Comment:
Assuming the file `modified` time is the published time, this data is published around 7:30 PM for the current day and next six days.